examples/tokenvm/storage/storage.go

```diff
@@ -357,9 +351,6 @@ func SetAsset(
 	binary.BigEndian.PutUint64(v[consts.Uint16Len+symbolLen+consts.Uint8Len+consts.Uint16Len+metadataLen:], supply)
 	copy(v[consts.Uint16Len+symbolLen+consts.Uint8Len+consts.Uint16Len+metadataLen+consts.Uint64Len:], owner[:])
 	b := byte(0x0)
```

Comment:
You no longer need `b`. You can remove it and remove the unnecessary alloc from this array (i.e. it should be a "boolean shorter").